R/splits_functions.R

```diff
@@ -36,29 +36,35 @@
 #'
 #' @export
 #'
-calcRate <- function(M, Aj, data_SS = NULL, days = NULL, searches = NULL){
+calcRate <- function(M, Aj, days = NULL, searches_carcass = NULL, data_SS = NULL){
  if (!is.null(data_SS) && ("SS" %in% class(data_SS))){
     days <- data_SS$days
-    searches <- data_SS$searches
+    unit <- rownames(Aj)
+    searches_carcass <- array(0, dim = c(x, length(days)))
```

Comment:
`x` is undefined... it's supposed to be the number of carcasses, right? so just `nrow(Aj)`?

Reply:
yup. good catch.